app/models/added_to_group_notification.rb

```diff
@@ -0,0 +1,17 @@
+class AddedToGroupNotification < ApplicationRecord
```

Comment:
Rubocop requests a documentation comment for this class (and for `RemovedFromGroupNotification`, too)